docs/userguide/userguide.md

```diff
@@ -1,8 +1,9 @@
-# User guide
+# User Guide
 
 ```{toctree}
 Campaigns <campaigns>
 Active Learning <active_learning>
+Async Workflow<async>
```

Comment:
Blank missing before < and I'd prefer the full "asynchronous" here

Reply:
changed
(the short version was deliberate to not make the sidebar too wide)